website/client-old/js/components/groupTasks/groupTasksController.js

```diff
@@ -0,0 +1,147 @@
+habitrpg.controller('GroupTasksCtrl', ['$scope', 'Shared', 'Tasks', 'User', function ($scope, Shared, Tasks, User) {
```

Comment:
The code in this file seems mostly duplicate, there's a way to share it with the tasks controller user for users?